addons/project/views/rating_views.xml

```diff
@@ -208,7 +258,11 @@
                 Let's wait for your customers to manifest themselves.
             </p>
         </field>
-        <field name="context">{'search_default_rating_last_30_days': 1, 'search_default_responsible': 1}</field>
+        <field name="context">{
+            'search_default_rating_last_30_days': 1,
+            'pivot_measures': ['rating', '__count__'],
```

Comment:
If you do this, then you force those measures. That is, if the user removes the `__count` measure and then change the group_by, the both measures will be activate again because the pivot uses the context.

Reply:
@xavierbol Removed 'prvot_meausres' from context.